plugin/pkg/scheduler/scheduler.go

```diff
@@ -222,7 +225,7 @@ func (sched *Scheduler) bind(assumed *v1.Pod, b *v1.Binding) error {
 	if err != nil {
```

Comment:
Updated this PR to call `FinishBinding` right after `bind` and log an error instead of returning. (thank you so much for all the comments!)